code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -162,6 +162,14 @@
 	amount_per_transfer_from_this = 42
 	list_reagents = list("salbutamol" = 10, "teporone" = 15, "epinephrine" = 10, "lavaland_extract" = 2, "weak_omnizine" = 5) //Short burst of healing, followed by minor healing from the saline
 
+/obj/item/reagent_containers/hypospray/autoinjector/emergency_nuclear
+	name = "emergency stabilization medipen"
+	desc = "A fast acting livesaving emergency autoinjector. Effective in combat situations, make by the syndicate for the syndicate.</span>"
+	icon_state = "stimpen"
+	volume = 15
+	amount_per_transfer_from_this = 15
+	list_reagents = list("perfluorodecalin" = 3, "teporone" = 3, "atropine" = 3, "stimulants" = 3, "mannitol" = 3)
```

Comment:
less than 5 u of stims will actually hurt them. I dont think the stimulants need to be here, unless you were looking for stimulative_agent